src/plugins/stake-plugins/generic/policyAdapters/EthereumKilnAdaptor.ts

```diff
@@ -33,6 +33,11 @@ export const makeEthereumKilnAdapter = (policyConfig: StakePolicyConfig<Ethereum
   const provider = new ethers.providers.FallbackProvider(rpcProviderUrls.map(url => new ethers.providers.JsonRpcProvider(url)))
   const integrationContract = KilnLiquid20A__factory.connect(contractAddress, provider)
 
+  // Metadata constants:
+  const metadataName = 'Kiln Pooled Staking'
```

Comment:
Do we usually localize metadata strings?

Reply:
It's a product name so we can leave it untranslated